### ISSUE TITLE
Move documentation over to the new website

### ISSUE THREAD

**Issue by martijn00** [MvvmCross/MvvmCross-Forms#37]
@PeterBurke We've launched a new website for MvvmCross. Could you move your excellent documentation over to that? You can find it at: http://mvvmcross.com/docs/xamarinforms

**Comment by PeterBurke** [comment]
Busy weekend but I will do 😁

Sent from my Windows Phone
________________________________

**Comment by PeterBurke** [comment]
I looked at this and yes I can create the relevant documentation. That would be at most 2 hours per day x 4 days per week. So that you know when to expect it, I will start by sending you a private estimate of time required.

**Comment by PeterBurke** [comment]
http://www.mvvmcross.com/docs/xamarinforms returns an error:

**Heroku | No such app. It has been doing this now for several weeks, I emailed Martijn about it but did not get a response.**

HeroKu is a cloud platform acquired by Sales force. The error above is described [here](https://support.dnsimple.com/articles/heroku-error-nosuchapp/).

So someone, possibly not even Tomasz or Martijn is responsible for this situation.

So I am closing this request, but you can reopen it if you can provide more details and you are not wasting my time.

**Comment by Cheesebaron** [comment]
remove www

**Comment by PeterBurke** [comment]
Yes well I can see the documentation. Where is the source: GitHub or HeroKu? If it is on HeroKu do I need write access?

**Comment by Cheesebaron** [comment]
Press the suggest edit button, is it that hard to understand?

**Comment by PeterBurke** [comment]
I have not tried that, but my question relates more to the nature of the person(s) who would then be controlling access, they seem to be faceless?

**Comment by Cheesebaron** [comment]
Controlling access to what?

**Comment by PeterBurke** [comment]
Can we take this offline?

I have looked at what is there and it seems that I wrote most of it, but it has been moved already. There are some gaps and the whole could do with a rewrite. Yes of course as a user if I needed to use a page I could offer a suggestion, but it is very unlikely to happen any time soon.

So basically I think that you should close this item.
